extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/DefaultTenantConfigResolver.java

```diff
@@ -26,12 +26,21 @@
     private volatile TenantConfigContext defaultTenant;
     private volatile Function<OidcTenantConfig, TenantConfigContext> tenantConfigContextFactory;
 
-    TenantConfigContext resolve(RoutingContext context) {
+    /**
+     * Resolve {@linkplain TenantConfigContext} which contains the tenant configuration and
+     * the active OIDC connection instance which may be null.
+     * 
+     * @param context the current request context
+     * @param create if true then the OIDC connection must be available or established
+     *        for the resolution to be successful
+     * @return
+     */
+    TenantConfigContext resolve(RoutingContext context, boolean create) {
         if (tenantConfigResolver.isAmbiguous()) {
             throw new IllegalStateException("Multiple " + TenantConfigResolver.class + " beans registered");
```

Comment:
I know this is not part of the PR, but this code should be in a PostCostruct method